public/ui.tsx

```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import ReactDOM, { Renderer } from 'react-dom'
 import { App as AppComponent } from 'ui/App'
 import { ConfigProvider } from 'ui/components/ConfigProvider'
+import { ApolloProvider } from 'ui/components/ApolloProvider'
 
 async function render(renderFunction: Renderer, App: typeof AppComponent) {
   renderFunction(
-    <ConfigProvider baseUrl={document.location.origin}>
-      <App />
+    <ConfigProvider {...window.FB_STATE.CONFIG}>
```

Comment:
I guess `FB_STATE` is for Firebase state? Makes my think of Facebook, lol. Hopefully, `CONFIG` is the only state value that needs passing this way, making the extra object unnecessary.